### PR TITLE
Add account number format

### PR DESCRIPTION
**Pull request by aathithya-zepto**
[Trello card](https://trello.com/c/V8eOqAWi/209-nz-api-docs-on-bban-format)

It is nice to have the account number format mentioned in NZ API docs.

Note: Didn't find a common place to add in the Bank Account format. So, added the format to following sections
1) Creating a new contact
2) Creating a KYC Agreement

### CODE REVIEW

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -1730,6 +1730,16 @@ paths:
           [KYC Trusted](https://help.zepto.money/en/articles/6217876-nz-kyc-trusted-general) article in our knowledge
           base for more information.
         </aside>
+
+        The format of the account number should be <B>BB-bbbb-AAAAAAA-SSS</B> where
+
+
+        - B is the bank number ( 2 digits) and b is the branch number(4 digits). Both constitute the prefix or the bank code.
```
Comment (kieranjogrady) [inline source/openapi3/split.yaml]:
```suggestion
        - B is the bank number (2 digits) and b is the branch number(4 digits). Both constitute the prefix or the bank code.
```

File: source/openapi3/split.yaml [hunk 2]
```diff
@@ -1730,6 +1730,16 @@ paths:
...
+
+        - A is the account number(7 digits) which is right justified
+
+        - S is the suffix(2 or 3 digits). When a bank displays the suffix as two digits, a leading zero is added to pad the suffix to three digits; i.e. BB-bbbb-AAAAAAA-SS becomes BB-bbbb-AAAAAAA-0SS.
```
Comment (zubin) [inline source/openapi3/split.yaml]:
```suggestion
        The format of the account number should be <B>BB-bbbb-AAAAAAA-SSS</B> where:


        - B is the bank number (2 digits) and b is the branch number (4 digits). Both constitute the prefix or the bank code.

        - A is the account number (7 digits) which is right justified.

        - S is the suffix (2-3 digits). When a bank displays the suffix as two digits, a leading zero is added to pad the suffix to three digits; i.e. BB-bbbb-AAAAAAA-SS becomes BB-bbbb-AAAAAAA-0SS.
```

File: source/openapi3/split.yaml [hunk 3]
```diff
@@ -1878,6 +1888,16 @@ paths:
...
```
Comment (zubin) [inline source/openapi3/split.yaml]:
```suggestion
        The format of the account number should be <B>BB-bbbb-AAAAAAA-SSS</B> where:


        - B is the bank number (2 digits) and b is the branch number (4 digits). Both constitute the prefix or the bank code.

        - A is the account number (7 digits) which is right justified.

        - S is the suffix (2-3 digits). When a bank displays the suffix as two digits, a leading zero is added to pad the suffix to three digits; i.e. BB-bbbb-AAAAAAA-SS becomes BB-bbbb-AAAAAAA-0SS.
```

**Review by zubin — APPROVED** [review]
I can't comment on the content, but have left some formatting suggestions.

**Review by kieranjogrady — APPROVED** [review]

File: source/openapi3/split.yaml (hunk 1 above)
Comment (kieranjogrady) [reply source/openapi3/split.yaml]: Ignore me, Zubin beat me to it.

**Pull request merged**